src/docker/mod.rs

```diff
@@ -19,9 +20,57 @@ pub struct DockerClient {
 
 impl DockerClient {
     /// Create a new Docker client
-    pub fn new() -> Result<Self, RuntimeError> {
+    pub async fn new() -> Result<Self, RuntimeError> {
         log::info!("Connecting to Docker daemon");
-        let client = bollard::Docker::connect_with_defaults()?;
+
+        log::info!("{}", std::env::var("DOCKER_HOST").unwrap());
```

Comment:
This causes a panic if `DOCKER_HOST` isnt set, i.e the thing we are checking for. unsure whether this log is even that useful tbh.